libraries/redcore/layouts/fields/rlist.php

```diff
@@ -50,7 +50,7 @@
 		<?php foreach ($data->options as $option) :?>
 				<option
 					value="<?php echo $option->value; ?>"
-					<?php if ((is_array($data->value) && in_array($option->value, $data->value)) || (!is_array($data->value) && $option->value == $data->value)):
+					<?php if ((is_array($data->value) && in_array($option->value, $data->value)) || (!is_array($data->value) && $option->value === $data->value)):
```

Comment:
this would take only 0 === 0 to be true, and "0" === 0 is false. Please look at this table:
http://stackoverflow.com/questions/80646/how-do-the-php-equality-double-equals-and-identity-triple-equals-comp